src/Symfony/Bundle/FrameworkBundle/Resources/config/error_renderer.php

```diff
@@ -32,7 +35,18 @@
                 service('logger')->nullOnInvalid(),
             ])
 
+        ->set('error_handler.error_renderer.cli', CliErrorRenderer::class)
+
+        ->set('error_handler.error_renderer.default', ErrorRendererInterface::class)
+            ->factory([SapiErrorRendererSelector::class, 'select'])
+            ->args([
+                service('error_renderer.cli'),
+                service('error_renderer.html'),
+            ])
+
         ->alias('error_renderer.html', 'error_handler.error_renderer.html')
-        ->alias('error_renderer', 'error_renderer.html')
+        ->alias('error_renderer.cli', 'error_handler.error_renderer.cli')
+        ->alias('error_renderer.default', 'error_handler.error_renderer.default')
+        ->alias('error_renderer', 'error_renderer.default')
```

Comment:
Do we need this alias?
```suggestion
        ->alias('error_renderer', 'error_handler.error_renderer.default')
```

Reply:
yes we need it, otherwise we'll get a circular reference when the serializer renderer is active.